### DISCUSSION TITLE
A result screenshot for a play from stable is alot more recognisable than one from lazer

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#17465] [Ideas] [answered]
The lack of the song background image in the results screen really hinders how quickly it is possible to identify what song a score corresponds do. When you look at lets say reddit. The most apparent and standout part of a posted score is the background image. Often vibrant and unique to a map. Maps like Freedom Dive, or Made of Fire have iconic, instantly recognisable background images.

But, currently in lazer the score card is clean and sleek but lacks the same instant recognisability as to what song/map it relates to. Combined with smaller song title text it makes it harder (at a glance) to know what song it relates to.

Example of what I mean below

Lazer:
![osu_2022-03-26_17-24-34](https://user-images.githubusercontent.com/45895074/160248885-c378f9b3-16ad-4170-9dd3-7166b843d5fc.jpg)

Stable:
![screenshot292](https://user-images.githubusercontent.com/45895074/160248874-bf0d4950-65ae-45ff-8b49-580f11e4b7cc.png)

Not exactly sure what current plans are for the skin editor. I'm assuming there will be more added to it over time (and quite possible what I'm going to suggest is already planned) but just wanted to comment on something I personally felt could be better.

Without making any massive pointless changes to the layout two things I have to suggest:
a. Only show personal result card (as a toggle). I personally don't really see the point of having the score card of someone else's play next to mine own in the results screen but I could imagine some people would like it. A toggle for this gives people more freedom of choice.
b. Blur Slider for results screen. Kinda self explanatory since a clear image is easy to recognise.

Would love to hear your opinion :)

**Comment by peppy** [comment, marked as answer]
Click the score card to show only your score.

Disabling blur will be a game wide setting in the future.
